### PR TITLE
Updating EIP 2515 for clarity.

### PR DESCRIPTION
**Pull request by MadeofTin**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - EIP header in EIPS/eip-2515.md does not match: 2512

File: EIPS/eip-2515.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-eip: 2512
```
Comment (axic) [inline EIPS/eip-2515.md:2]: How was this merged with the wrong number in the first place?

File: EIPS/eip-2515.md [hunk 2]
```diff
@@ -53,17 +55,20 @@ if (BLOCK_HEIGHT  <= LAST_FORK_HEIGHT + DIFFICUTLY_FREEZE_DIFFERENCE) :
...
-Block height is very easy to predict and evaluate within the system. This removes the effect of the Difficulty Bomb on block time, simplifying the block time targeting mechanism. The addition of an increase in the difficulty was added after feedback that the game theory of the mechanism did not reliably result in .
+Block height is very easy to predict and evaluate within the system. This removes the effect of the Difficulty Bomb on block time, simplifying the block time targeting mechanism. The addition of an increase in the difficulty was added after feedback that the game theory of the mechanism did not reliably result in planned obsolescence of the chain. It was still theoretically possible that a chain could continue in a zombie state with miners self-regulating hashrate to keep the chain at a semi-functional state. While the game theory tended towards miners reaching blocktimes too fast for the network to propagate, the addition of a steady increase in difficulty removes this possibility and is therefore more secure.
 
 https://twitter.com/quentinc137/status/1227110578235330562
 
+__Rational behind 0.001% Increase__
```
Comment (axic) [inline EIPS/eip-2515.md:69]: Why not use markdown headings?

```suggestion
### Rational behind the 0.001% Increase
```

File: EIPS/eip-2515.md [hunk 3]
```diff
@@ -53,17 +55,20 @@ if (BLOCK_HEIGHT  <= LAST_FORK_HEIGHT + DIFFICUTLY_FREEZE_DIFFERENCE) :
...
+The algorithm is not purely linear, but slowly increases exponentially every block by .01% of the previous. It still needs to be determined if a pure linear function is prefered, or which value of the scalar if it is exponential.
```
Comment (axic) [inline EIPS/eip-2515.md:70]:
```suggestion
The algorithm is not purely linear, but slowly increases exponentially every block by .01% of the previous. It still needs to be determined if a pure linear function is preferred, or which value of the scalar if it is exponential.
```

prefered -> preferred typo, which makes the build fail

**Comment by axic** [comment]
All these fields are broken:
```
created: 2020/02/10
requires (*optional):
replaces (*optional):
```

File: EIPS/eip-2515.md (hunk 1 above)
Comment (MadeofTin) [reply EIPS/eip-2515.md:2]: No idea. It confuses me.

File: EIPS/eip-2515.md [hunk 4]
```diff
@@ -1,29 +1,27 @@
...
-eip: 2512
-title: Implement Difficulty Freeze
+eip: 2515
+title: Update Difficulty Bomb with a Freezing Mechanism
 author: James Hancock (@madeoftin)
 discussions-to: https://ethereum-magicians.org/t/eip-2515-replace-the-difficulty-bomb-with-a-difficulty-freeze/3995
 status: Draft
 type: Standards Track
 category: Core
 created: 2020/02/10
```
Comment (axic) [inline EIPS/eip-2515.md:9]:
```suggestion
created: 2020-02-10
```

File: EIPS/eip-2515.md [hunk 5]
```diff
@@ -1,29 +1,27 @@
...
+The current difficulty bombs' effect on the Block Time Targeting mechanism is rather complex to model, and it has both appeared when it was not expected (Muir Glacier) and negatively affected users and miners when they are not the target (in the case of delaying forks due to technical difficulties). Users are affected as a function of the usability of the chain is affected by increased block times. Miners are affected by a reduction in block rewards due to the increase in block time. Both of these groups are unable on their own to address the difficulty bomb. In the case of the Difficulty Freeze it is more predictiable and so knowing when to make the change is readily apparent at all times.
```
Comment (axic) [inline EIPS/eip-2515.md:24]: There is a single difficulty bomb:

```suggestion
The current difficulty bomb's effect on the Block Time Targeting mechanism is rather complex to model, and it has both appeared when it was not expected (Muir Glacier) and negatively affected users and miners when they are not the target (in the case of delaying forks due to technical difficulties). Users are affected as a function of the usability of the chain is affected by increased block times. Miners are affected by a reduction in block rewards due to the increase in block time. Both of these groups are unable on their own to address the difficulty bomb. In the case of the Difficulty Freeze it is more predictable and so knowing when to make the change is readily apparent at all times.
```

File: EIPS/eip-2515.md [hunk 6]
```diff
@@ -53,17 +53,20 @@ if (BLOCK_HEIGHT  <= LAST_FORK_HEIGHT + DIFFICUTLY_FREEZE_DIFFERENCE) :
...
-Block height is very easy to predict and evaluate within the system. This removes the effect of the Difficulty Bomb on block time, simplifying the block time targeting mechanism. The addition of an increase in the difficulty was added after feedback that the game theory of the mechanism did not reliably result in .
+Block height is very easy to predict and evaluate within the system. This removes the effect of the Difficulty Bomb on block time, simplifying the block time targeting mechanism. The addition of an increase in the difficulty was added after feedback that the game theory of the mechanism did not reliably result in planned obsolescence of the chain. It was still theoretically possible that a chain could continue in a zombie state with miners self-regulating hashrate to keep the chain at a semi-functional state. While the game theory tended towards miners reaching blocktimes too fast for the network to propagate, the addition of a steady increase in difficulty removes this possibility and is therefore more secure.
 
 https://twitter.com/quentinc137/status/1227110578235330562
 
+### Rational behind the 0.001% Increase
```
Comment (axic) [inline EIPS/eip-2515.md:67]:
```suggestion
### Rationale behind the 0.001% Increase
```

**Comment by axic** [comment]
@MadeofTin you need to click the "Resolve conflicts" button and manually fix the issues, otherwise this can not be merged:
Screenshot

**Comment by lightclient** [comment]
Pinging @MadeofTin.

**Pull request closed without merging**